test/Microsoft.ML.CodeGenerator.Tests/ApprovalTests/ConsoleCodeGeneratorTests.cs

```diff
@@ -45,7 +46,7 @@ public ConsoleCodeGeneratorTests(ITestOutputHelper output) : base(output)
             }
         }
 
-        [Fact]
+        [NotAppleSiliconFact("Model Builder not supported on Apple Silicon")]
```

Comment:
I don't understand this.

1. What is "Model Builder" in this context? Are you referring to the VS Model Builder?
2. What makes it not supported?

Reply:
I am referring to the VS Model Builder. From my understanding the `CodeGenerator` is only used for the Model Builder when it is creating the the cs projects for the user. 

Model Builder is not supported on anything but Windows currently, I assumed it was some limitation on the other platforms but that is just an assumption. Since it works on normal OSX maybe I am wrong.

I don't remember the exact error this gives. Let me re-run the tests and get that. I think it was a null error though.

Reply:
> From my understanding the CodeGenerator is only used for the Model Builder when it is creating the the cs projects for the user.
> Model Builder is not supported on anything but Windows currently

That may be true today, but there is no reason for the code generator not to be able to run across any platform - OS or architecture. It should work the same regardless.

Reply:
Ok, so I am re-running these and they are passing locally on my M1. So its possible that the failures were due to the helix machine issues they were having, or something else not set quite right on the helix machines. I will remove this and make sure it runs in the helix machines.